### ISSUE TITLE
Substring matches?

### ISSUE THREAD

**Issue by stephanbuys** [Keats/tera#229]
Hi, I had a cursory look through the docs and the code (v0.11), but couldn't spot something to enable substring matches `assert!("xyz".contains("x") == true)`

**Comment by Keats** [comment]
Indeed there isn't anything for that :o.
I think a `containing` test that works on string/array/maps should be added. If someone want to do it, please do it on the 0.11 branch or i'll do it myself sometimes next week.

**Comment by stephanbuys** [comment]
Hi, I tried to test this but no real luck, I tried matching in `string` values in a list of `maps`, also a `for` loop over an array of `strings`, am I missing something perhaps? I used the same syntax as you used in the test:
```
{% for k in stringList %}
{% if k is containing("somestring") %}
Yes!
{% else %}
No!
{% endif %}
{% endfor %}  
```

Not sure how to test the negative.

**Comment by Keats** [comment]
That's because I had forgotten to actually register it in Tera >_> Try again after pulling and it should be fine.

To do the negative: `{% if not k is containing("somestring") %}`

**Comment by Keats** [comment]
Fixed as of 0.11.0-beta, stable release will come once `pest` releases 1.0
